### PR TITLE
- Minor changes to AJM and macros

### PR DESCRIPTION
**Pull request by christianneurohr**

### CODE REVIEW

File: doc/sec_paper/AbelJacobiMap.tex [hunk 1]
```diff
@@ -73,18 +72,17 @@
...
   and call {\em big period matrix} the concatenated matrix \\
-  \todo (CN) without comma? $(\OA \, \OB)$? $\in \C^{g \times 2g}$ or $\C^{2g \times g}$?
   \begin{equation}
-      \Omega = (\OA,\OB)
+      \Omega = (\OA \,\OB) \in \C^{g\times 2g}
   \end{equation}
   such that $\Lambda = \Omega\Z^{2g}$.
 
-  If one takes as basis of differentials the dual basis of
+  If one takes as basis of differentials the dual basis of\,,
```
Comment (pascalmolin) [inline doc/sec_paper/AbelJacobiMap.tex:80]: I have a question: why is your tex filled with small spaces like that? is it your editor? I think we should remove them all.

File: doc/sec_paper/AbelJacobiMap.tex [hunk 2]
```diff
@@ -73,18 +72,17 @@
...
   the loops $α_i$ the matrix becomes
   \begin{equation}
-      \OA^{-1}\Omega = (I_g,\tau)
+      \OA^{-1}\Omega = (I_g\, \tau)\,,
```
Comment (pascalmolin) [inline doc/sec_paper/AbelJacobiMap.tex:83]: So what are the arguments for and against comma?

**Review by pascalmolin — COMMENTED** [review]
OK, just two questions

**Comment by christianneurohr** [comment]
- I'm used to put some spacing behind math expressions, because i think it
looks nicer... if you want we can remove them...
- For me the notation ( A B ) means its a block matrix from A and B. ( A,
B) means its a family of matrices?

By the way: I'm searching for the function abc_unit_root (you wrote it
appearantly), but its nowhere to be found (only in the documentary)


2016-12-19 17:02 GMT+01:00 pascalmolin <notifications@github.com>:

**Comment by pascalmolin** [comment]
- tex is supposed to handle math spacing nicely in math expressions, there are always counterexamples but as a general rule I do not like adding more (and I guess editors would not either)
- since I do not like spaces very much, I prefer the comma because a couple of matrices is just the same as a concatenated matrix while a product is completely different

**Pull request merged**